src/components/JumpToPage.js

```diff
@@ -0,0 +1,107 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import {  
+  InputBase,  
+  MenuItem,  
+  Select,  
+  Toolbar,  
+  Typography
+} from '@material-ui/core';
+import { withStyles } from '@material-ui/core/styles';
```

Comment:
Method Higher-order component(`withStyles`)  looks better than method Hooks(`makeStyles`).

Especially in the case of functional component

`withStyles` to `makeStyles`

However, it is not an unconditional task.
It has nothing to do with behavior | logic.

Reply:
Great catch! I first had the component as a class but then decided to make it a hook, however, I must have gotten distracted and forgot the withStyles/makeStyles part. Kind of interesting that everything still worked fine. I'll push an update here shortly though.